.travis.yml

```diff
@@ -2,5 +2,7 @@ language: node_js
 node_js:
   - 0.8
   - 0.10
+  - 0.12
+  - iojs
 before_install:
   - npm install -g npm@~1.4.6
```

Comment:
@andrewbranch oh, sorry - I was totally shure that this was deleted :smile_cat: 

@simonjosefsson we needed `npm v1.4.6` in order to let the tests pass for `node v0.8.0` too while pulling in some new libraries. This could probably make it into another PR where we should discuss about support for the (very, very) old `node v0.8.0` in general.